requirements.txt

```diff
@@ -27,5 +27,5 @@ clang-format==19.1.0
 
 pytlsd@git+https://github.com/iago-suarez/pytlsd.git@37ac583
 deeplsd@git+https://github.com/cvg/DeepLSD.git@88c589d
-gluestick@git+https://github.com/cvg/GlueStick.git@0f28efd
+-e git+https://github.com/cvg/GlueStick.git@0f28efd#egg=gluestick
```

Comment:
Hey. What justifies this? Do we need to update it in the main branch instead?

Reply:
GlueStick subfolders (e.g. `models`) could not be found otherwise.
But yes you are right, that's something we should probably change in the main branch.

Reply:
I see. Did not test it through. Should we fix it in the GlueStick repo instead?

Reply:
Yes, you're right. I'll do it now and revert the changes to requirements here.